### PR TITLE
[MPSCore] Xcode11.1 GM Binding

### PR DESCRIPTION
**Pull request by prmallic**
Xcode11.1 GM Diff - https://github.com/xamarin/xamarin-macios/wiki/MPSCore-iOS-xcode11.1-GM
Added missing 'offset' selector

### CODE REVIEW

File: src/metalperformanceshaders.cs [hunk 1]
```diff
@@ -2089,6 +2093,10 @@ interface MPSVector {
 		[TV (11,3), Mac (10,13,4), iOS (11,3)]
```
Comment (whitneyschmidt) [inline src/metalperformanceshaders.cs]: nit: spacing for version #

**Comment by VincentDondain** [comment]
@prmallic I can help you with that (:

**Comment by VincentDondain** [comment]
Mmh indeed, looks like Xcode 11.1(.2) is not listing older 13.X simulators... https://forums.developer.apple.com/thread/123347

**Comment by VincentDondain** [comment]
_@prmallic note that if it comes from the 13.1 API diffs, it's fair to use that version for the availability attribute instead of being too smart and go for the older version (which might be incorrect)._

**Comment by prmallic** [comment]
Device test on iOS13.0 failed. So bumping the version to 13.1

**Pull request merged**